### DISCUSSION TITLE
wat happen

### DISCUSSION THREAD

**Discussion by switchupcb** [bwmarrin/discordgo#1403] [General]
when I took my programming hiatus in february, discordgo had 74 issue.
now it has 90 issue.
wat happen man

![discordgos-future](https://github.com/bwmarrin/discordgo/assets/81384235/95ce6e01-e519-40c0-9236-06d343a721c4)

we can do something before its too late.

[disgo](https://github.com/switchupcb/disgo) add all new features from last 150 days in less than 6 hours
wat now
look at stars in sky

jk
few more things to do until its 100% completion
but come on why discordgo not even use [dasgo](https://github.com/switchupcb/dasgo)?

**Comment by FedorLap2006** [comment]
There's a problem of clutter, yes. Currently there's a bit more focus put into features, rather than issues.
However some major ones get fixed from time to time.
